backend/packages/Upgrade/src/api/DTO/ExperimentDTO.ts

```diff
@@ -201,6 +201,25 @@ class ConditionPayloadValidator {
   public decisionPoint?: string;
 }
 
+class OldConditionPayloadValidator {
```

Comment:
You can create a base abstract class for `OldConditionPayloadValidator` and `ConditionPayloadValidator`.
It is not a big deal.